### PR TITLE
Adding inspect support for endpointGroup

### PR DESCRIPTION
**Pull request by gaurav-dalvi**
Adding inspect support for endpointGroup object

### CODE REVIEW

**Comment by shaleman** [comment]
@gaurav-dalvi just FYI on the process:
Can you please submit the pull request for contivmodel repo first and then vendor it in here once its merged?

**Comment by gaurav-dalvi** [comment]
@shaleman : Thanks for reminding. Here is PR for contivModel code :
https://github.com/contiv/contivmodel/pull/36

**Comment by gaurav-dalvi** [comment]
Once this is merged I will update this PR !

File: netmaster/objApi/apiController.go [hunk 1]
```diff
@@ -657,6 +657,57 @@ func (ac *APIController) EndpointGroupUpdate(endpointGroup, params *contivModel.
...
+	epgCfg.StateDriver = stateDriver
+	epgID := endpointGroup.Config.GroupName + ":" + endpointGroup.Config.TenantName
+	fmt.Printf("EPGID is %s", epgID)
+	if err := epgCfg.Read(epgID); err != nil {
+		log.Errorf("Error fetching endpointGroup from mastercfg: %s", epgID)
+		return err
+	}
+
+	endpointGroup.Oper.ExternalPktTag = epgCfg.ExtPktTag
+	//network.Oper.NumEndpoints = nwCfg.EpCount
```
Comment (shaleman) [inline netmaster/objApi/apiController.go]: We should populate this field based on number of endpoints in endpointGroup.Oper.Endpoints array

File: netmaster/objApi/objapi_test.go [hunk 2]
```diff
@@ -617,8 +631,11 @@ func TestTenantDelete(t *testing.T) {
 	checkCreateNetwork(t, false, "tenant1", "net1", "data", "vlan", "10.1.1.1/16", "10.1.1.254", 1, "", "")
 	checkCreateNetwork(t, false, "tenant1", "net2", "data", "vlan", "20.1.1.1/16", "20.1.1.254", 2, "", "")
 	checkCreateEpg(t, false, "tenant1", "net1", "group1", []string{}, []string{})
+	checkInspectEpg(t, false, "tenant1", "group1", 1)
```
Comment (shaleman) [inline netmaster/objApi/objapi_test.go]: These tests are a good start.
But, they can not verify if epg-inspect is showing the endpoints correctly.
We should start adding integration tests for these cases to emulate an endpoint create and verify they show up in inspect.
Basic framework for integration tests are part of PR #478. Can we wait for #478 to be merged and add integration tests for epg-inspect as well?

Comment (gaurav-dalvi) [inline netmaster/objApi/objapi_test.go]: Yes that sounds good. I will wait for that PR to be merged and then update test cases for endpointGroup inspect.

File: netmaster/objApi/apiController.go (hunk 1 above)
Comment (gaurav-dalvi) [inline netmaster/objApi/apiController.go]: Updated now !

File: netmaster/objApi/objapi_test.go (hunk 2 above)
Comment (shaleman) [inline netmaster/objApi/objapi_test.go]: @gaurav-dalvi PR #478 has been merged, if you want to add integration test for this too..

File: netmaster/objApi/apiController.go [hunk 3]
```diff
@@ -657,6 +657,59 @@ func (ac *APIController) EndpointGroupUpdate(endpointGroup, params *contivModel.
...
+	// Get the state driver
+	stateDriver, err := utils.GetStateDriver()
+	if err != nil {
+		return err
+	}
+
+	epgCfg := &mastercfg.EndpointGroupState{}
+	epgCfg.StateDriver = stateDriver
+	epgID := endpointGroup.Config.GroupName + ":" + endpointGroup.Config.TenantName
+	fmt.Printf("EPGID is %s", epgID)
```
Comment (abhinandanpb) [inline netmaster/objApi/apiController.go]: can you please remove this log

**Pull request closed without merging**